packages/gatsby-source-contentful/src/__tests__/__snapshots__/normalize.js.snap

```diff
@@ -2963,7 +2963,7 @@ Array [
       "displayField": "title",
       "id": "c6XwpTaSiiI2Ak2Ww0oi6qa",
       "internal": Object {
-        "contentDigest": "755626d87af0751f341ff2e4ec082ed9",
+        "contentDigest": undefined,
```

Comment:
ohh ohh

Reply:
There are like 20 ` "contentDigest": undefined` in this snapshot. I bet somewhere `node` is not passed through properly.

Reply:
Or the test data is incorrect?